cn_indian_payroll/cn_indian_payroll/page/payroll_configuratio/payroll_configuratio.js

```
@@ -903,422 +903,978 @@ frappe.pages['payroll-configuratio'].on_page_load = function(wrapper) {
+							salary_component_array.push({
+								"component": rowData.salary_component,
+								"abbr":rowData.abbr,
+								"type":rowData.component_type,
+								"formula":rowData.formula,
+								"condition":rowData.condition
+							});
+
+							if(rowData.is_arrear)
+							{
+								salary_component_array.push({
+									"component": rowData.salary_component+"(Arrear)",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)